src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -51,7 +51,7 @@ public static void ProcessLogFile(string filePath, Solution solution, string too
                     {
                         // They're opening a v1 log, so we need to transform it.
                         // Ask if they'd like to save the v2 log.
-                        MessageDialogCommand response = PromptToSaveProcessedLog(Resources.TransformV1_DialogMessage);
+                        MessageDialogCommand response = promptOnLogConversions ? PromptToSaveProcessedLog(Resources.TransformV1_DialogMessage) : MessageDialogCommand.No;
```

Comment:
>MessageDialogCommand [](start = 145, length = 20)

I agree this change in default is useful.

Reply:
Yep, works great.

---
In reply to: [440859508](https://github.com/microsoft/sarif-visualstudio-extension/pull/143#discussion_r440859508) [](ancestors = 440859508)